nni/algorithms/hpo/dngo_tuner/dngo_tuner.py

```diff
@@ -0,0 +1,117 @@
+import random
+import torch
+import numpy as np
+
+from nni.tuner import Tuner
+import nni.parameter_expressions as parameter_expressions
+from torch.distributions import Normal
+from pybnn import DNGO
```

Comment:
We should add `pybnn` into an optional dependency required by DNGO. See how PPO and BOHB does that.

Reply:
ok, will change it this weekend